scripts/get_code_check_paths.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$#" != 1 ]; then
+  echo "Invalid number of arguments, expected 1: lint/format";
+  exit 1;
+fi;
+
+if [ "$1" == "lint" ]; then
+  find . -name '*.py'
```

Comment:
I've just run it locally, and the linter complained about files in the virtual environment (.venv). Maybe we can duplicate source of paths in favor of simplicity?

Reply:
Nice catch. First, I thought we could use paths specified in gitignore to exclude them from the search but it looked too hacky and weird. I think specifying the paths explicitly is simpler and clearer.

Reply:
[explicitly specify paths to look for python files](https://github.com/software-mansion/protostar/pull/1295/commits/8e6123f43cbf50d807047a164327afc6256b6713)